pkg/queue/prometheus_stats_reporter_test.go

```diff
@@ -164,16 +164,23 @@ func TestReporterReport(t *testing.T) {
 			if err != nil {
 				t.Errorf("Something went wrong with creating a reporter, '%v'.", err)
 			}
+			// Make the value slightly more interesting, rather than microseconds.
+			reporter.startTime = reporter.startTime.Add(-5 * time.Second)
 			reporter.Report(test.concurrency, test.proxiedConcurrency, test.reqCount, test.proxiedReqCount)
 			checkData(t, requestsPerSecondGV, test.expectedReqCount)
 			checkData(t, averageConcurrentRequestsGV, test.expectedConcurrency)
 			checkData(t, proxiedRequestsPerSecondGV, test.expectedProxiedRequestCount)
 			checkData(t, averageProxiedConcurrentRequestsGV, test.expectedProxiedConcurrency)
+			checkData(t, processUptimeGV, 5.0)
 		})
 	}
 }
 
-func checkData(t *testing.T, gv *prometheus.GaugeVec, wanted float64) {
+// Except for uptime everything else is integers, so this precision is
+// good enough for the tests.
+const precision = 0.1
```

Comment:
nit

Can we move this **const** to the starting line where already few constants declared

Reply:
Sure.